src/main/java/ca/uhn/fhir/jpa/starter/common/StarterJpaConfig.java

```diff
@@ -448,5 +462,45 @@ public static IServerConformanceProvider<?> calculateConformanceProvider(IFhirSy
 			throw new IllegalStateException();
 		}
 	}
+
+	private void registerLocationHierarchyTypes(RestfulServer restfulServer, DaoRegistry daoRegistry) {
```

Comment:
Add the Hardcoded strings to Constants or use the existing Constants.

Reply:
Done